source/isaaclab/isaaclab/sensors/frame_transformer/frame_transformer.py

```diff
@@ -35,6 +35,32 @@
 # import logger
 logger = logging.getLogger(__name__)
 
+# Regex pattern for environment path parsing
+# Matches: env_0/, env_.*/, etc. and captures the path after
+_ENV_PATH_PATTERN = re.compile(r"env_[^/]+/(.*)")
```

Comment:
This seems the only occurance of this pattern. Might move to the function itself?